variables.tf

```diff
@@ -142,3 +142,10 @@ variable "priority" {
   type    = number
   default = null
 }
+
+variable "timeout_h" {
```

Comment:
How about `auto_resolve_time` regardless of the variable name that DD chose to use.